docs/approvals/jira-service-management/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Jira Service Management Integration
+description: Octopus Deploy can integrate with your Jira Service Management instance for deployment 
+control using Change Requests/Issues
+position: 10
+---
+
+:::warning
+The Jira Service Management (JSM) Integration feature is only available as an EAP for version 
+**2022.3.
+12101** or later and it requires a specific feature license to use it. To request a 
+license register for the [JSM Early Access Program](https://octopusdeploy.
+typeform.com/servicenow-eap)
```

Comment:
should be a link to the JSM eap, but you probably already were aware of this

Reply:
add a task to the ticket for that - but need to work out what to do about that first!